sql/mysql-compatibility.md

```diff
@@ -155,6 +154,10 @@ Due to its distributed nature, workloads that are single-threaded may perform wo
 
     When TiDB is in the execution of loading data, by default, a record with 20,000 rows of data is seen as a transaction for persistent storage. If a load data operation inserts more than 20,000 rows, it will be divided into multiple transactions to commit. If an error occurs in one transaction, this transaction in process will not be committed. However, transactions before that are committed successfully. In this case, a part of the load data operation is successfully inserted, and the rest of the data insertion fails. But MySQL treats a load data operation as a transaction, one error leads to the failure of the entire load data operation.
     
+### Views
+
+Views in TiDB are currently non-updatable. We plan to add this functionality at a later date.
```

Comment:
Would `non-updatable and non-insertable` be better?

Reply:
Yes.  I have changed it, thx